packages/libs/error-stack/src/ext/future.rs

```diff
@@ -128,6 +142,48 @@ pub trait FutureExt: Future + Sized {
     /// Adds new contextual information to the [`Frame`] stack of a [`Report`] when [`poll`]ing the
     /// [`Future`].
     ///
+    /// This behaves like [`attach_printable()`] but will not be shown when printing the [`Report`].
```

Comment:
It's convenient to have this in the same place but I think we're introducing maintenance burden and a chance for doc-comments to deviate. We should probably just say "Refer to `attach_printable()` on [\`Report\`] for more information, or something like that